sensor_frame:
  z: -1.65
  z_min: -2.4

polar_grid:
  range:
    min: 10.0
    max: 40.0 # 83.0
  cell:
    size: 0.25 # 0.25
    max_height: 0.15
  segments: 10 # 240

cart_grid:
  cell:
    size: 0.25

semantic:
  edge_detection:
    perform: false
    min: 5
    max: 120
    kernel: 3

ransac:
  tolerance: 0.2
  iterations: 50

